tests/js/sentry-test/reactTestingLibrary.tsx

```diff
@@ -20,6 +21,11 @@ import {instrumentUserEvent} from '../instrumentedEnv/userEventIntegration';
 import {initializeOrg} from './initializeOrg';
 
 interface ProviderOptions {
+  /**
+   * Mounts the GlobalDrawer, and its provider.
+   * Must be specified excplitly since .toBeEmptyDOMElement assertions will fail otherwise
+   */
+  includeDrawer?: boolean;
```

Comment:
i was thinking more like another state that hides `<GlobalDrawer />` might even behave differently in tests since the animation matters a lot less. Like it renders once you open a drawer

```suggestion
  /**
   * Mounts the GlobalDrawer, and its provider.
   * Must be specified excplitly since .toBeEmptyDOMElement assertions will fail otherwise
   */
  includeDrawer?: boolean;
```